README.md

```diff
@@ -3,7 +3,7 @@
 # Portfolio: setup and mobile version skeleton
 > My Eighty  Project Portfolio - mobile menu.
 
-![screenshot](./imgs/app-image.png)
+![screenshot](./imgs/popup.jpg)
 
```

Comment:
Keep professionalism in your projects. 
- Kindly provide a description about the features implemented in your branch.